### PR TITLE
Fix Nightblade weapon types and elusive mod persistance

### PR DESCRIPTION
**Pull request by LocalIdentity**
Fix Nightblade to not work if the player is dual wielding with either a Sword, Axe or Mace
Fix elusive mod persisting if checkbox was ticked and the source to generate elusive was taken away

### CODE REVIEW

File: Modules/ConfigOptions.lua [hunk 1]
```diff
@@ -718,23 +718,23 @@ return {
 		modList:NewMod("Condition:KilledUniqueEnemy", "FLAG", true, "Config", { type = "Condition", var = "Combat" })
 	end },
 	{ var = "conditionHaveArborix", type = "check", label = "Do you have Iron Reflexes?", ifCond = "HaveArborix", tooltip = "This option is specific to Arborix",apply = function(val, modList, enemyModList)
-		modList:NewMod("Condition:HaveIronReflexes", "FLAG", true, "Config", { type = "Condition", var = "Combat" })
-		modList:NewMod("Keystone", "LIST", "Iron Reflexes", "Config")
+		modList:NewMod("Condition:HaveIronReflexes", "FLAG", true, "Config", { type = "Condition", var = "Combat" }, { type = "Condition", var = "HaveArborix" })
+		modList:NewMod("Keystone", "LIST", "Iron Reflexes", "Config", { type = "Condition", var = "HaveArborix" })
```
Comment (NickRyder) [inline Modules/ConfigOptions.lua]: are these changes related?

File: Modules/ConfigOptions.lua [hunk 2]
```diff
@@ -718,23 +718,23 @@ return {
...
-		modList:NewMod("Keystone", "LIST", "Avatar of Fire", "Config")
+		modList:NewMod("Condition:HaveAvatarOfFire", "FLAG", true, "Config", { type = "Condition", var = "Combat" }, { type = "Condition", var = "HaveVulconus" })
+		modList:NewMod("Keystone", "LIST", "Avatar of Fire", "Config", { type = "Condition", var = "HaveVulconus" })
 	end },
-	{ var = "conditionHaveManaStorm", type = "check", label = "Do you have Manastorm's Lightning Buff?", ifCond = "HaveManaStorm", tooltip = "This option is enable Manastorm's Lightning Damage Buff\nWhen you cast a Spell, Sacrifice all Mana to gain Added Maximum Lightning Damage\nequal to 25% of Sacrificed Mana for 4 Seconds", apply = function(val, modList, enemyModList)
+	{ var = "conditionHaveManaStorm", type = "check", label = "Do you have Manastorm's Lightning Buff?", ifCond = "HaveManaStorm", tooltip = "This option will enable Manastorm's Lightning Damage Buff\nWhen you cast a Spell, Sacrifice all Mana to gain Added Maximum Lightning Damage\nequal to 25% of Sacrificed Mana for 4 Seconds", apply = function(val, modList, enemyModList)
```
Comment (NickRyder) [inline Modules/ConfigOptions.lua]: Also these

File: Modules/ConfigOptions.lua (hunk 1 above)
Comment (LocalIdentity) [reply Modules/ConfigOptions.lua]: Nope, these are meant to be part of another pull request

File: Modules/ConfigOptions.lua (hunk 2 above)
Comment (LocalIdentity) [reply Modules/ConfigOptions.lua]: Also meant to be a part of a different PR
